### DISCUSSION TITLE
Uncaught SyntaxError: Unexpected token '<'

### DISCUSSION THREAD

**Discussion by aasokvolvo** [vercel/next.js#59879] [Help]
### Summary

I am using Next 14 and created a sample create next app. I am able to build and run the app locally without any fail. But, when I deploy the same to server, I am getting Uncaught SyntaxError: Unexpected token '<' in several pages and a warning as  well "The resource ***/_next/static/media/c9a5bc6a7c948fb0-s.p.woff2 was preloaded using link preload but not used within a few seconds from the window's load event. Please make sure it has an appropriate `as` value and it is preloaded intentionally."

I think it is not able to load the static files

In my server.js config part looks like below:
app.use(express.static(".next/static"));

app.get("/*", function (req, res) {
  res.sendFile(
    path.join(__dirname, "./.next/server/app/index.html"),
    function (err) {
      console.log("inside server.js file");
      console.log(__dirname);
      console.log(path);
      if (err) {
        res.status(500).send(err);
      }
    }
  );
});

Could you please help me with this?

**Comment by ghost** [comment]
Same issue, i followed tutorial from next.js website [https://nextjs.org/learn/dashboard-app/setting-up-your-database](https://nextjs.org/learn/dashboard-app/setting-up-your-database). Everything works fine in local dev, and when i deployed to vercel, the build and deployment was successful but when loading the page the chunks can't be parsed:

![image](https://github.com/user-attachments/assets/27264367-9568-431b-9174-fbb8ea5360c6)

I've been searching for couple hours but find no solution
